web/admin/src/App.js

```diff
@@ -21,23 +21,25 @@ export default class App extends Component {
   componentDidMount() {
     fbc.signinAdmin()
     .then(() => {
-      client.getUsers().then(attendees => this.setState({attendees: attendees.sort(sortPlayers)}))
+      client.getUsers().then(attendees => {this.setState({attendees: attendees.sort(sortPlayers)}) 
```

Comment:
Since this is being turned into a multi-line function, let's not leave the first line of the function on the same line as the `{`